extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -142,14 +162,27 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
       paramList.push({name: 'prev_fmts', value: sharedStateParams.prevFmts});
     }
 
+    if (this.isResponsive_ &&
```

Comment:
@dvoytenko because here it wants a custom formula for the height
```
height = width / 1.91 + 120
```

so the `i-amphtml-sizer` will have `paddingTop`:

```
paddingTop: calc(52.35% + 120px);
```